tools/rvn/OfflineOperations.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.IO;
+using Raven.Server.ServerWide;
+using Raven.Server.Utils;
+using Sparrow;
+using Voron;
+using Voron.Impl.Compaction;
+
+namespace rvn
+{
+    // TODO: Implement correctly get-key, put-key, trust, encrypt, decrypt
+    internal class OfflineOperations
+    {
+        private const string SecretKeyEncrypted = "secret.key.encrypted";
+        
+        public static string GetKey(string srcDir)
```

Comment:
@iftahbe What is going on here?
@aviviadi The code make no sense, please have it reviewed